options/linux/include/utmp.h

```diff
@@ -2,6 +2,8 @@
 #ifndef  _UTMP_H
 #define  _UTMP_H
 
+#include <stdint.h>
```

Comment:
Why do we need this?

Reply:
We should rather replace the uses by `__mlibc_uint32` (etc.) from `bits/types.h`.